scripts/bootstrap-aio.yml

```diff
@@ -17,6 +17,25 @@
   hosts: localhost
   user: root
   tasks:
+    - name: Ensure the hosts file is templated appropriately
+      copy:
+        content: |
+          127.0.0.1 localhost aio1
+          127.0.1.1 aio1.openstack.local aio1
+          # The following lines are desirable for IPv6 capable hosts
+          ::1 ip6-localhost ip6-loopback
+          fe00::0 ip6-localnet
+          ff00::0 ip6-mcastprefix
+          ff02::1 ip6-allnodes
+          ff02::2 ip6-allrouters
+          ff02::3 ip6-allhosts
+        dest: /etc/hosts
+        backup: yes
+
+    - name: Ensure hostname is set
+      hostname:
+        name: aio1
```

Comment:
Are you doing this because the nodepool hostname is too long, or another reason?

Reply:
Yes, check the commit message: https://github.com/rcbops/rpc-openstack/pull/3062/commits/94ff2eb5a4c114001ddfd30b7e811a3da6d272df